docs/applications/plex.mdx

```diff
@@ -103,6 +103,109 @@ import SystemdTabs from '../snippets/systemdtabs.mdx';
 
 <SystemdTabs service="plexmediaserver"/>
 
+## Plex Nginx Configuration (optional)
```

Comment:
I'd move this entire chapter all the way to the bottom, I would assume `Connect to other clients` might be more popular

Reply:
10-4